pages/user.js

```diff
@@ -0,0 +1,46 @@
+import Cookies from "cookies";
+import Iron from "@hapi/iron";
+import { ENC_KEY } from "./api/users";
```

Comment:
For separation and readability import from external package could be grouped aswell as import of internal resources 

```
import Cookies from "cookies";
import Iron from "@hapi/iron";
import Link from "next/link";

import { ENC_KEY } from "./api/users";
```